src/app/python/core.py

```diff
@@ -559,7 +570,39 @@ def _task_create_archive(self, path, caller, response):
 
     # Export file
     response.put(archive_path)
-    
+
+  @task_time(microseconds=1)
+  def _task_write_to_disk(self, caller):
```

Comment:
Right, this was a just quick feature based on the "exec based on live version" writing to disk code.